test/integration/test_payroll_process_scenarios/indice.js

```diff
@@ -0,0 +1,143 @@
+   */
+  it('POST /multiple_payroll/:id/ Payroll Management: Continuity of Work For Employees Amelia Rose Thornton', () => {
+    return agent.post('/multiple_payroll_indice/parameters/')
+      .send(paramsPayEnvelope)
+      .then((res) => {
+        expect(res).to.have.status(201);
+
+        return agent.get('/multiple_payroll_indice/')
+          .query(queryParameters);
+      })
+      .then((res) => {
+        // Olivier Benjamin Hensley
+        // EM.TE.1266
+        // Hiring Date : January 1, 2015
+        // Seniority**: 10 years (as of January 1, 2025)
+
+        const dataOlivierBenjaminHensley = res.body.employees.find(item => item.employee_reference === 'EM.TE.1266');
+
+        // ### Job Details
+        // - Grade : ATB1 – First-Class Office Assistant
+        // - Position : Nurse
+        // - Base Index : 200
+        // - Responsibility Index (linked to the function): 60
+        // #### Adjusted Base Index Calculation
+        // - Annual growth rate : 5%
+        // - Years of seniority : 10 years
+        // > Adjusted Index = Base Index × (1 + growth rate) ^ years of seniority
+        // > Adjusted Index = 200 × (1 + 0.05) ^ 10 ≈ 200 × 1.62889 ≈ 325.78
+        // Approximate Adjusted Index : 326
+
+        const rubricAdjustedBaseIndex = dataOlivierBenjaminHensley.rubrics.find(
+          item => item.rubric_abbr === 'BASE',
+        );
+
+        expect(rubricAdjustedBaseIndex.rubric_value).to.equal(326);
+
+        // ### Employee Work Summary
+        // - Worked Days : 23
+        // - Additional Days : 4
+        // - Total Days Counted : 27
+
+        // ### Step 1: Daily Index Calculation
+        // We calculate the daily index by dividing the Base Index and Responsibility Index
+        // by the number of standard worked days:
+        // > Daily Index = (Base Index + Responsibility Index) / Worked Days = (326 + 60) / 23 = 16.782
+
+        // ### Step 2: Adjusted Index Calculation
+        // Multiply the Daily Index by the Total Days Worked (including additional days) to get the adjusted index:
+        // > Adjusted Index = Daily Index × Total Days = 16.782 × 27 ≈ 453.13
+
+        const rubricDailyIndexCalculation = dataOlivierBenjaminHensley.rubrics.find(
+          item => item.rubric_abbr === 'BJ-PRS',
+        );
+
+        const dailyIndexCalculation = Math.round(rubricDailyIndexCalculation.rubric_value * 100) / 100;
+        expect(dailyIndexCalculation).to.equal(453.13);
+
+        // ### Additional Indexed Benefits
+        // - Night Shift Bonus – Indexed : 2
+        // - Overtime – Indexed : 3
+        // > Base Salary and Indexed Benefits = 453.130435 + 2 + 3 = 458.130435
+
+        const rubricBaseSalaryIndexed = dataOlivierBenjaminHensley.rubrics.find(
+          item => item.rubric_abbr === 'BAV-IND',
+        );
+
+        const baseSalaryIndexed = Math.round(rubricBaseSalaryIndexed.rubric_value * 10000) / 10000;
```

Comment:
Why was this necessary?  It looks like you are modifying the data returned by the server before testing it.